app/mci/pkg/gcp/loadbalancer/loadbalancersyncer_test.go

```diff
@@ -80,7 +80,8 @@ func TestCreateLoadBalancer(t *testing.T) {
 		}
 		return true, ret, nil
 	})
-	if err := lbc.CreateLoadBalancer(&ing); err != nil {
+	force := true
```

Comment:
We should test with force=false by default.
force=true can mask errors.

Also we dont really need this extra var here
If its for documentation, you can write it as `CreateLoadBalancer(&ing, false /* force */)`